app/assets/stylesheets/common.scss

```diff
@@ -945,6 +945,76 @@ img.trace_image {
   .node, .way, .relation {
     margin-left: 25px;
   }
+
+  .svg_icon {
+    overflow: hidden;
+    display: inline-block;
+    margin-left: -25px;
+    width: 25px;
+    height: 18px;
+    /*rtl:ignore*/ transform: translate(2px, 0px);
```

Comment:
What's this `translate` for?

Old css background image above, new svg image below:
![image](https://github.com/user-attachments/assets/ab0a218a-0592-4708-9626-c9ddb09ffe63)

Reply:
The purpose of the translate is to improve the positioning of SVGs. Like I've written below, that's good for the majority of new SVGs.

The old css PNG images will be replaced by SVGs in later PRs, by which time this issue will become obsolete. Until then, some fine tuning could still be done in follow up PRs.

Reply:
highway=turning_circle has been replaced by an SVG symbol.